services/imagery/package.json

```diff
@@ -8,7 +8,7 @@
     "start": "node ./bin/start-service",
     "build": "babel src --out-dir lib",
     "build-msg": "mkdir -p lib && pbjs -t static-module --es6 --keep-case -o src/messages.js src/messages/*.proto",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
Solves issue of test flakiness mentioned in #59. Jest by default runs tests in parallel, so multiple tests were modifying the sqlite file at the same time (hence the unexplained behavior of rolling back a transaction that wasn't underway).